docs/user-interface/graphics/blendmodes.md

```diff
@@ -0,0 +1,170 @@
+---
+title: ".NET MAUI Graphics: Blend modes"
+description: "The .NET MAUI graphics library enables different compositing operations to be specified by the BlendMode property."
+ms.date: 07/30/2021
+---
+
+# .NET MAUI Graphics: Blend modes
```

Comment:
Don't forget to come back and alter these titles based on our discussions over the next day or so.